DolbyIO/DolbyIO.Android.xml

```diff
@@ -0,0 +1,66 @@
+<?xml version="1.0" encoding="utf-8"?>
+<root xmlns:android="http://schemas.android.com/apk/res/android">
+  <init>
+    <log text="DolbyIO plugin init"/>
+  </init>
+
+  <gradleProperties>
+    <insert>
+org.gradle.jvmargs=-Xmx1536M --add-exports=java.base/sun.nio.ch=ALL-UNNAMED --add-opens=java.base/java.lang=ALL-UNNAMED --add-opens=java.base/java.lang.reflect=ALL-UNNAMED --add-opens=java.base/java.io=ALL-UNNAMED --add-exports=jdk.unsupported/sun.misc=ALL-UNNAMED
+ANDROID_BUILD_TOOLS_GRADLE_VERSION=com.android.tools.build:gradle:7.3.1
+ANDROID_TOOLS_BUILD_GRADLE_VERSION=com.android.tools.build:gradle:7.3.1
+    </insert>
+  </gradleProperties>
+
+  <soLoadLibrary>
+    <loadLibrary name="dolbyio_comms_sdk_android_cppsdk" 
+      failmsg="XML! libdolbyio_comms_sdk_android_cppsdk.so library not loaded and required!" />
+  </soLoadLibrary>
+
+  <baseBuildGradleAdditions>
+    <insert>
+      allprojects {
+        repositories {
+          repositories {
+            flatDir {
```

Comment:
Can I specify a local AAR without using Maven and without using flatDir? flatDir generates warnings.

Reply:
for curent versions, flatDir is fine, I don't see any other way right now. Once the cppsdk starts shipping the dependency to sonatype, it will be mandatory to remove those blocks anyway (and for intermediate versions, install in local maven repository : ~/.m2/repository/io/dolby etc... so that we avoid using aars)